app/models/exercise.rb

```diff
@@ -58,8 +58,8 @@ def self.for(track_slug, exercise_slug)
     self.git_important_files_hash = Git::GenerateHashForImportantExerciseFiles.(self) if self.git_important_files_hash.blank?
```

Comment:
I could not use `Exercise::UpdateHashForImportantExerciseFiles` here, as that calls `.update` and at this point it can't (no record yet). I also could not move this to `after_create`, as the `git_important_files_hash` has to be non-empty.

Not using the command doesn't really matter that much here, as the additional functionality it offers (updating in a transaction, updating of the solutions with the same hash) don't really matter here (there are no solutions when creating the exercise).